readme/local-development.md

```diff
@@ -24,15 +24,23 @@ To use [Drupal VM](http://www.drupalvm.com/) with a Drupal project that is gener
         brew install php56 git composer ansible drush
         brew cask install virtualbox vagrant
 
-1. Create & boot the VM, install Drupal.
+1. Create & boot the VM.
 
         blt vm
+
+1. Install Drupal.
+
+        vagrant ssh
         blt setup
 
-1. Login to Drupal `drush @[project.machine_name].local uli`, where [project.machine_name] is the value that you set in blt.yml.
+1. Login to Drupal.
+
+        drush uli
 
 There are also other changes you can make if you choose to match the Acquia Cloud server configuration more closely. See Drupal VM's example configuration changes in Drupal VM's `examples/acquia/acquia.overrides.yml` file.
 
+Note: With a Drupal VM setup, BLT expects all commands (with a couple of exceptions), to be executed within the VM. To SSH into the VM, simply run `vagrant ssh` as you did in the "Install Drupal" step above.
```

Comment:
What exceptions? `blt vm`?

Reply:
and `blt vm:nuke`, I believe. We can nix that if you think it will cause confusion.

Reply:
Yeah I think it's confusing. I'd be more specific. Like, with the exception of `blt vm:*` commands...